src/components/ForecastSummary.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import WeatherIcon from "react-icons-weather";
+import "../styles/ForecastSummary.css";
+
+function ForecastSummary(props) {
+  const { date, description, icon, temperature, onSelect } = props;
+  const formattedDate = new Date(date).toDateString();
+  return (
+    <div className="forecast-summary" data-testid="forecast-summary">
+      <div className="forecast-summary__date">{formattedDate}</div>
+      <div className="forecast-summary__icon" data-testid="forecast-icon">
+        <WeatherIcon name="owm" iconId={icon} />
+      </div>
+      <div className="forecast-summary__temperature">
+        {temperature.max}&deg;C
+      </div>
+      <div className="forecast-summary__description">{description}</div>
+      <button
+        type="button"
+        onClick={() => onSelect(date)}
+        className="forecast-summary__button"
+      >
+        More details
+      </button>
+    </div>
+  );
+}
+
+export default ForecastSummary;
+
+ForecastSummary.propTypes = {
```

Comment:
Its convention to declare your `propTypes` block above the export component declaration, this should be moved there